### ISSUE TITLE
DM the registering user via the App's home on onboarding

### ISSUE THREAD

**Issue by jcpsantiago** [jcpsantiago/thearqivist#52]

**Comment by tjb** [comment]
@jcpsantiago would you be open to someone newer to clojure contributing? if so, i can take a look at this!

**Comment by jcpsantiago** [comment]
Hey thanks for your interesting in giving a hand! If you're ok with code review, and getting and giving feedback you're more than welcome to work on it :) 

Let me update this ticket with more context so you're set up for success.

**Comment by tjb** [comment]
Yes I am very into the code review process and would love feedback to learn & grow! 

As a note, I tried to start my repl in cursive and run `(start)` but it mentions that it cannot find the symbol in the namespace. unsure if you came across this issue before or not but just an FYI. 

There is a holiday this week in the US so I will continue to tinker to get the local dev environment running / wait for an update to the ticket.

**Comment by jcpsantiago** [comment]
@practicalli-john made some changes to the README to make it clearer there are some dependencies on aliases, which sound like you don't have setup.

We tested it in a brand new repo and it worked :)

**Comment by jcpsantiago** [comment]
Raising the priority of this one https://github.com/jcpsantiago/thearqivist/issues/2
